docs/ilm-how-to.asciidoc

```diff
@@ -1,18 +1,191 @@
+[discrete]
+[[index-lifecycle-policies-default]]
+=== Default policies
 
-To view and edit these index lifecycle policies in {kib},
-select *Stack Management* / *Index Lifecycle Management*.
-Search for `apm`.
+The table below describes the default index lifecycle policy applied to each APM data stream.
+Each policy includes a rollover and delete definition:
 
-See {ref}/getting-started-index-lifecycle-management.html[manage the index lifecycle] for more information.
+* **Rollover**: Using rollover indices prevents a single index from growing too large and optimizes indexing and search performance. Rollover, i.e. writing to a new index, occurs after either an age or size metric is met.
+* **Delete**: The delete phase permanently removes the index after a time threshold is met.
+
+[cols="1,1,1",options="header"]
+|===
+|Data stream
+|Rollover after
+|Delete after
+
+|`traces-apm`
+|30 days / 50 gb
+|10 days
+
+|`traces-apm.rum`
+|30 days / 50 gb
+|90 days
+
+|`metrics-apm.profiling`
+|30 days / 50 gb
+|10 days
+
+|`metrics-apm.internal`
+|30 days / 50 gb
+|90 days
+
+|`metrics-apm.app`
+|30 days / 50 gb
+|90 days
+
+|`logs-apm.error`
+|30 days / 50 gb
+|10 days
+
+|===
+
+The APM index lifecycle policies can be viewed in {kib}.
+Navigate to *Stack Management* / *Index Lifecycle Management*, and search for `apm`.
+
+[discrete]
+[[data-streams-custom-policy]]
+=== Configure a custom index lifecycle policy
+
+This tutorial explains how to apply a custom index lifecycle policy to the
+`traces-apm` data stream with the `default` namespace.
+
+[discrete]
+[[data-streams-custom-one]]
+=== Step 1: View data streams
+
+The **Data Streams** view in {kib} shows you the data streams,
+index templates, and index lifecycle policies associated with a given integration.
+
+. Navigate to **Stack Management** > **Index Management** > **Data Streams**.
+. Search for `apm` to see all data streams associated with APM data.
+. Select the `traces-apm-default` data stream to view its associated index template and ILM policy.
+As you can see, the data stream follows the <<apm-data-streams-naming-scheme>> and starts with its type, `traces-`.
++
+[role="screenshot"]
+image::images/data-stream-overview.png[Data streams info]
+
+[discrete]
+[[data-streams-custom-two]]
+=== Step 2: Create an index lifecycle policy
+
+. Navigate to **Stack Management** > **Index Lifecycle Policies**.
+. Click **Create policy**.
+
+Name your new policy. I've chosen `custom-traces-apm-default-policy`.
+Customize the policy to your liking, and when you're done, click **Save policy**.
+
+[discrete]
+[[data-streams-custom-three]]
+=== Step 3: Create a component template
+
+To apply your new index lifecycle policy to a data stream,
+and to ensure the policy continues to be applied in future versions,
+you must create a component template.
+The component template name must start with `.`, follow the data stream naming scheme,
```

Comment:
Where did you get the `.` part from? AFAIK they do not have a dot prefix.

Reply:
The dot (and these docs) are adapted from a tutorial that @hop-dev wrote.

Reply:
@axw Thanks for pointing this out, I think this is a mistake in my original guide, you're right the component templates don't have a dot prefix. 

@joshdover I can't see any record of us discussing having a dot prefix on the namespace specific component templates, I think we may be able to remove it? If so I'll get a PR in for the docs.